benefits/static/css/styles.css

```diff
@@ -540,6 +545,34 @@ footer .footer-links a {
 }
 
 @media (max-width: 992px) {
+  /* Mobile With Image */
+
+  .with-image main .main-row .col-lg-6.image {
+    background: center bottom url("/static/img/ridertappingbankcard.png")
```

Comment:
Why is this image path repeated? Can't we use `background-image` once and change whatever else is needed in the media queries?